### DISCUSSION TITLE
Supabase + Sveltekit webapp tutorial Logout not working as expected.

### DISCUSSION THREAD

**Discussion by vivasvan1** [supabase/supabase#12944] [Questions]
# Pressing signout will signout successfully but wont redirect to the login page automatically

## pressing signout is supposed to redirect to the login page which is not working.

I am pretty sure that logout redirect is not working because of use:enhance in the form.

Another problem which might be related is: 
also instead of the magic_link if you use the username password 
that also doesn't seem to redirect to /account as well.

Something like this
```
<Auth
	supabaseClient={data.supabase}
	redirectTo={`${data.url}/logging-in?redirect=/`}
	showLinks={false}
	appearance={{ theme: ThemeSupa, style: { input: 'color: #fff' } }}
/>
```
This also doesnt redirect you to /account automatically.

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. First follow all the steps here https://supabase.com/docs/guides/getting-started/tutorials/with-sveltekit
2. go to localhost:5137
3. after login and on /account
4. click logout.

## Expected behavior

Should go back to path /

## System information

- OS: Ubuntu 20.04
- Version of supabase-js: 2.10.0
- Version of Node.js: 19.7.0

## Additional context

I only want login with email and password to redirect to home page.

**Comment by GaryAustin1** [comment]
You really should seek help first in Supabase Discord or here in Discussions before reporting a bug.

`redirectTo` is sent to the auth calls being made that use `redirectTo`.  `signOut()` does not use `redirectTo` so this will have no effect.
`signInWithPassword()` will also not redirect.

`redirectTo` is used when the auth requests sends an email link or using a 3rd party OAuth call.

You probably need to handle your in browser/javascript redirect if you want those calls to go to another page when finished.

Note I don't use auth-ui myself, so can't help you debug it further.

I'm going to move this to discussions right now as not clear this is a bug, versus maybe needs to be a better documentation request.
